src/twisted/test/test_compat.py

```diff
@@ -585,11 +444,7 @@ def test_unicode(self):
         """
         C{compat.unicode} is C{str} on Python 3, C{unicode} on Python 2.
         """
-        if _PY3:
-            expected = str
-        else:
-            expected = unicode
-        self.assertIs(unicodeCompat, expected)
+        self.assertIs(unicodeCompat, str)
```

Comment:
This looks fine, but in the docstring above, you might want to remove mention of Python 2, since
you are removing the test for Python 2.